CMakeLists.txt

```diff
@@ -16,6 +16,17 @@ include(cmake/curl.cmake)
 include(cmake/tinyxml2.cmake)
 include(cmake/zlib.cmake)
 
+if(NOT MSVC)
+    include(third_party/cpp_rsc/cmake/modules/cpp_resource.cmake)
+    if(NOT ANDROID AND NOT IOS)
+        # If not cross-compiling, we can build cpp_rsc.
+        add_subdirectory(third_party/cpp_rsc/src)
+    else()
+        # For cross-compilation, re-use the binary from default.
+        set_property(GLOBAL PROPERTY CPPRSC_CMD ${CMAKE_CURRENT_BINARY_DIR}/../default/third_party/cpp_rsc/src/cpp_rsc)
```

Comment:
I don't get where that is used :/

Reply:
It's used in https://github.com/orex/cpp_rsc/blob/928ad84e11d768b1c6a3aa427277234b49b53cb5/cmake/modules/cpp_resource.cmake#L71.

Reply:
I see. And `add_resource` and `link_resource_file` come from cpp_rsc as well.